zk-token-sdk/src/instruction/batched_range_proof/mod.rs

```diff
@@ -54,7 +54,7 @@ impl BatchedRangeProofContext {
     fn new_transcript(&self) -> Transcript {
         let mut transcript = Transcript::new(b"BatchedRangeProof");
         transcript.append_message(b"commitments", bytes_of(&self.commitments));
-        transcript.append_message(b"bit-legnths", bytes_of(&self.bit_lengths));
+        transcript.append_message(b"bit-lengths", bytes_of(&self.bit_lengths));
```

Comment:
@samkim-crypto , will this typo fix cause any issues downstream?

Reply:
Discussed in DMs. Since it will need to be backported, we will fix separately: https://github.com/solana-labs/solana/pull/34468
@GoodDaisy , please remove this change from your PR.

Reply:
Yes, it will break compatibility with 1.17. I created https://github.com/solana-labs/solana/pull/34468 that addresses this part of the typo so that it can be backported.